packages/cairn/cairn.1.0/opam

```diff
@@ -0,0 +1,45 @@
+opam-version: "2.0"
+synopsis: "A derivation explorer and logger for menhir parser"
+description:
+  "A small library that takes a parser generated by menhir, logs its execution and provides an interactive terminal user interface to explore it."
+maintainer: ["Vincent Penelle"]
+authors: ["Vincent Penelle"]
+license: "GPL-2.0-or-later"
+homepage: "https://github.com/VincentPenelle/cairn"
+bug-reports: "https://github.com/VincentPenelle/cairn/issues"
+depends: [
+  "menhirSdk"
+  "menhirLib"
+  "ocaml"
+  "dune" {>= "2.9"}
+  "menhir"
```

Comment:
This needs a suitable lower bounds on both menhir and the OCaml version.
Eg 
```suggestion
 "ocaml" {>= "4.08"}
  "menhir" {>= "20230608"}
```

4.08 is implied by the use of `String.split_on_char` (since 4.04) and Int module since 4.08. Not sure about the `Zed_string` warning?

Reply:
Indeed, many thanks @tmcgilchrist (and sorry for not having though about that myself, it is my first package contribution, I didn't think to check the version of the functions I use).
For the Zed_string warning, I've searched quickly on zed homepage, I do not see versions for the functions in the API. Maybe let's see if the warning persist after your changes (I accepted to quickly and don't find anymore the log). At worst, would it be a good idea to put the version I used as a lower bound?